config/config.go

```diff
@@ -117,19 +117,17 @@ func (cfg *config) init(env *AppConfig) error {
 }
 
 func (cfg *config) SetupCompleted() bool {
-	// TODO: remove AlbyUserIdentifier and hasLdkDir checks after 2025/01/01
+	// TODO: remove hasLdkDir check after 2025/01/01
```

Comment:
Just curious, what changed after v1.6.0? Why can't we remove both checks at once?

Reply:
In 1.6.0 we made the Alby Account optional, and the existence of the Alby Account was used to check if the user had setup their node (as it was the last thing that happened after setting up the node)

When we made the Alby Account optional, we instead introduced the `NodeLastStartTime` config entry which we use to detect if they ever went through the setup.

So for backward compatibility, we had these old checks. But now, the Alby Account can be connected before the node is setup, so I had to remove that one. I was going to remove both, but I decided to keep the LDK directory check in as this is more critical to ensure the user does not go through the setup again (and wipe their old data)